gem/templates/base/comments/comment.html

```diff
@@ -14,10 +14,10 @@
           {% trans "Big Sister" %}
         {% endif %}
     {% elif node.user_name.lower == 'anonymous' %}
-      {{node.user_name}}
+      {% trans settings.commenting.CommentingSettings.commenting_anonymous %}
```

Comment:
we are not able to translate a value of variable that is not set in the template. For Pootle you would need the text that you want to be translated in the template. so the above syntax is not right.